frontend/src/pages/Fight.jsx

```diff
@@ -137,6 +161,7 @@ function Fight({ hero, bossesList }) {
           </button>
         </div>
       </div>
+      <HeroLossModal Modal={Modal} open={open} close={close} isOpen={isOpen} />
```

Comment:
looks like you're not using open, close nor isOpen in HeroLossModal: you can remove them from props. and why not:

    <Modal>
      <HeroLossModal />
    </Modal>

so you won't need props anymore